packages/documentation-site/patternfly-docs/content/developer-resources/rtl-handbook.md

```diff
@@ -0,0 +1,44 @@
+---
+id: Right to left handbook
```

Comment:
```suggestion
id: Right-to-left handbook
```
Or should this be called "Bidirectionality handbook"? Not sure which term is most recognizable to devs